packages/synapse-react-client/src/style/components/_cards.scss

```diff
@@ -76,6 +77,18 @@ $text-color-lighter: #898989;
       color: white !important;
       text-decoration-color: white !important;
     }
+    .MuiButton-root {
+      border-color: white !important;
+      color: white !important;
+      height: 24px;
+
+      &:hover {
+        background-color: map.get(SRC.$primary-color-palette, 700);
+      }
+      &:active {
+        background-color: map.get(SRC.$primary-color-palette, 800);
+      }
+    }
```

Comment:
Add header card style for buttons. There are already styles for MUI Links, so it seemed more appropriate to extend the existing stylesheet.